gtests/net/mptcp/common/set_sysctls.py

```diff
@@ -1,21 +1,21 @@
 #!/usr/bin/python
 
-import commands
+import subprocess
 import os
 import sys
 
 pppid = int(os.popen("ps -p %d -oppid=" % os.getppid()).read().strip())
 filename = '/tmp/sysctl_restore_%d.sh' % pppid
 
 restore_file = open(filename, 'w')
-print >> restore_file, '#!/bin/bash'
+restore_file.write('#!/bin/bash')
 
 for a in sys.argv[1:]:
   sysctl = a.split('=')
 
   # save current value
   cur_val = commands.getoutput('cat ' + sysctl[0])
```

Comment:
do you not need to modify this line too? I see you removed `commands` to switch to `subprocess` but you also need to modify this bit, no?

(note: maybe you don't need to use this helper? See below)

Reply:
> do you not need to modify this line too? I see you removed `commands` to switch to `subprocess` but you also need to modify this bit, no?
> 
> (note: maybe you don't need to use this helper? See below)

ENOCOFFEE here. I submitted an old version of this script - with bugs included. I'll update the PR soon.